frontend/src/components/LinkListTile.vue

```diff
@@ -18,16 +18,20 @@ SPDX-License-Identifier: Apache-2.0
           </template>
           {{appTitle}} is not running for hibernated clusters
         </v-tooltip>
-        <a v-else :href="sanitizeUrl(url)" target="_blank" rel="noopener">{{urlText}}</a>
+        <external-link v-else :url="url"></external-link>
```

Comment:
If we really want to use the external link here we also should remove the sanitizeUrl mixin.